### ISSUE TITLE
Permission denied on OS X

### ISSUE THREAD

**Issue by lyptt** [keybase/client#3401]
When attempting to copy a file to my private folder in KBFS Finder ended up not responding and everything went to hell. After forcing a reboot (the only way to actually get my machine functional again), I can no longer access `/keybase`.

My log id: ec76c86d4c98c2e39549c21c

**Comment by lyptt** [comment]
Turns out Keybase is now completely broken on my machine. Even purging all the files Keybase has chucked on my filesystem (why no uninstaller?) didn't help.

**Comment by strib** [comment]
Hi @lyptt - what version of OS X are you running?  Any chance it's the new macOS beta?  (I see reference to a version "10.12" in one of our logs.) If so, we don't support it yet, and there are known issues.

I took a look at your logs, and there is some behavior that I haven't seen before in OS X installations (specifically, file system operations are taking 60 seconds and then getting canceled, instead of the usual 19 second timeout that we force on file system operations on OS X).

I can't yet explain what happened to your original FS operations, it seems like they just start freezing.  But after the reboot it looks like you can't mount the FS anymore.

**Comment by lyptt** [comment]
Yup I'm on the public beta. Didn't see a reference on the download page to it not being supported.

**Comment by strib** [comment]
@malgorithms: can we put a warning on the download page about macOS?  Seems like lots of people are starting to use it.

**Comment by idyll** [comment]
@strib and @malgorithms I think part of the problem is that you're calling "OS X" the new name "macOS" which implies to people that you are supporting 10.12 when you don't.

It looks like Fuse 3.4.0 will support macOS but I imagine there's more to it than that...

**Comment by lyptt** [comment]
That's right - I assumed that because it said 'macOS' on the download page that it supported 10.12. Interestingly the filesystem seemed to work fine until I tried copying a file into it.

**Comment by malgorithms** [comment]
Ahh, I was overly optimistic!

I just changed the wording around the download page and there's now a warning about macOS.

I'm also going to make an internal ticket for the app letting you know when you try to run it on an unsupported OS version. That needs to work not just for betas but also for old versions of OSX/Windows that we don't support.

**Comment by rsaeks** [comment]
Just adding another "me as well" to the issue. Just an idea for deployment down the road: using a macOS .pkg format allows for version checking during the install. That should be one way to prevent the app from being installed on an unsupported system.
